k8s/configmap.yaml

```
@@ -0,0 +1,53 @@
+kind: ConfigMap
+metadata:
+  name: blog-app-config
+  namespace: blog-app
+  labels:
+    app: blog-app
+    component: config
+data:
+  # Server Configuration
+  NODE_ENV: "production"
+  PORT: "5000"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)